src/kw_db.sh

```diff
@@ -209,7 +212,11 @@ function select_from()
     return 22 # EINVAL
   fi
 
-  sqlite3 -init "$KW_DB_DIR/pre_cmd.sql" -cmd "$pre_cmd" "$db_path" -batch "SELECT $columns FROM $table;"
+  query="SELECT $columns FROM $table ;"
+  if [[ -n "${order_by}" ]]; then
+    query="SELECT $columns FROM $table ORDER BY ${order_by} ;"
+  fi
+  sqlite3 -init "$KW_DB_DIR/pre_cmd.sql" -cmd "$pre_cmd" "$db_path" -batch "$query"
```

Comment:
Can we just use `select_from`?

Reply:
Sorry if I'm  missing something, but I don't think we can do this here as this is part of the definition of `select_from` itself.